examples/sample-app/updateregistry.sh

```diff
@@ -3,5 +3,5 @@
 # registry service ip+port.  You must have started openshift and deployed
 # the docker-registry service before running this.
 # The new template file produced is template.json.
-REGISTRY_IP=$(openshift cli get services docker-registry -o template --template="{{ .portalIP}}:{{ .port }}")
+REGISTRY_IP=$(osc get services docker-registry -o template --template="{{ .portalIP}}:{{ .port }}")
```

Comment:
I think you can delete this whole script...the templates no longer contain the registry service IP.